data.lua

```diff
@@ -21,7 +21,7 @@ end
 if settings.startup["miniloader-enable-chute"].value then
   local standard_enabled = settings.startup["miniloader-enable-standard"].value
   settings.startup["miniloader-enable-standard"].value = true
-  create_miniloader("chute-", "", {"logistics"}, {r=0.5,g=0.5,b=0.5}, "underground-belt")
+  create_miniloader("chute-", "", {"logistics"}, {r = 0.5, g = 0.5, b = 0.5, a = 0.82}, "underground-belt")
```

Comment:
Could you use [util.color](https://github.com/wube/factorio-data/blob/master/core/lualib/util.lua#L65-L78) for consistency with below?